README.md

```diff
@@ -53,16 +67,26 @@ See image below:
 
 ![image](https://user-images.githubusercontent.com/22883860/123566757-97a43a00-d814-11eb-9e3e-1d2468145e3d.png)
 
-### Running test1
+### Running a test
 
-Go to 'Run>Configuration per file...' and check the **The following directory** under **Working directory settings** and specify the root of the repository with no quotes.
+Go to 'Run>Configuration per file...' and check the **The following directory** under **Working directory settings** and specify the root of the repository with no quotes. Doe this for each Python test file (usually one per test folder)
```

Comment:
These instructions are IDE specific so it would be good to remind the reader which IDE this is for.
Or you could write it IDE-agnostically. Whichever one you prefer

Spelling here:
```suggestion
Go to 'Run>Configuration per file...' and check the **The following directory** under **Working directory settings** and specify the root of the repository with no quotes. Do this for each Python test file (usually one per test folder)
```